definitions/variable/environment/biodiversity.yaml

```
@@ -1,62 +1,113 @@
+      to human pressures. The BII is an estimated percentage of the original number of species that remain and 
+      their abundance in any given area, despite human impacts. The main difference between MSA and BII is that 
+      every unit of land is given equal weight in MSA, whereas BII gives more weight to species-rich areas.
 - Terrestrial Biodiversity|Biodiversity Intactness Index|Biodiversity Hotspots:
     description: Terrestrial biodiversity in biodiversity hotspot landscapes measured
       with Biodiversity Intactness Index (BII)
     unit: '%'
+    min: 0
+    max: 100
     sdg: 15
     weight: Land Cover
     tier: 3
+    notes: The Biodiversity Intactness Index (BII) summarises the change in ecological communities in response 
+      to human pressures. The BII is an estimated percentage of the original number of species that remain and 
+      their abundance in any given area, despite human impacts. The main difference between MSA and BII is that 
+      every unit of land is given equal weight in MSA, whereas BII gives more weight to species-rich areas.
 - Terrestrial Biodiversity|Biodiversity Intactness Index|Key Conservation Landscapes:
     description: Terrestrial biodiversity in biodiversity hotspot and intact forest landscapes
       measured with Biodiversity Intactness Index (BII)
     unit: '%'
+    min: 0
+    max: 100
     sdg: 15
     weight: Land Cover
     tier: 3
+    notes: The Biodiversity Intactness Index (BII) summarises the change in ecological communities in response 
+      to human pressures. The BII is an estimated percentage of the original number of species that remain and 
+      their abundance in any given area, despite human impacts. The main difference between MSA and BII is that 
+      every unit of land is given equal weight in MSA, whereas BII gives more weight to species-rich areas.
 - Terrestrial Biodiversity|Biodiversity Intactness Index|Areas Outside Key Conservation and Cropland Landscapes:
     description: Terrestrial biodiversity in areas outside biodiversity hotspot,
       intact forest & cropland landscapes measured with Biodiversity Intactness Index (BII)
     unit: '%'
+    min: 0
+    max: 100
     sdg: 15
     weight: Land Cover
     tier: 3
+    notes: The Biodiversity Intactness Index (BII) summarises the change in ecological communities in response 
+      to human pressures. The BII is an estimated percentage of the original number of species that remain and 
+      their abundance in any given area, despite human impacts. The main difference between MSA and BII is that 
+      every unit of land is given equal weight in MSA, whereas BII gives more weight to species-rich areas.
 - Terrestrial Biodiversity|Shannon Crop Diversity Index:
     description: Crop diversity measured by the Shannon index accounting for crop richness
       and abundance
-    unit:
+    unit: 'index'
```

A dimensionless timeseries should have no unit (instead of the word "index"), see [the docs](https://docs.ece.iiasa.ac.at/standards/variables.html#units).
```suggestion
    unit:
```